WebRoot/css/login.css

```diff
@@ -90,6 +90,30 @@ select#client:focus {
 	float:none;
 }
 
+.chooseTFAMethod,
+.chooseTFAMethodBack {
+	float: inline-end;
```

Comment:
this is supported in mostly new browsers only https://developer.mozilla.org/en-US/docs/Web/CSS/float
can this give issue in older browsers?

Reply:
Thank you @silentsakky for pointing it out. I confirmed the style was not applied on IE11 (MS Edge, IE mode). It will be addressed in another ticket to check/update messages and styles.